frontend/components/common/label.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Label = ({ text, color, bg }) => (
+  <LabelWrapper color={color} bg={bg}>
+    {text}
+  </LabelWrapper>
+);
+
+Label.propTypes = {
+  text: PropTypes.string.isRequired,
+  color: PropTypes.string,
+  bg: PropTypes.string,
+};
+Label.defaultProps = {
+  color: 'white',
+  bg: 'red',
+};
+
+const LabelWrapper = styled.div`
+  display: inline-box;
```

Comment:
혹시 `inline-block`인가요?
```suggestion
  display: inline-block;
```